### PR TITLE
sdk: Provide more details about the error communicating with adsd3500

### PR DESCRIPTION
**Pull request by dNechita**

### CODE REVIEW

File: sdk/src/connections/target/adsd3500_sensor.cpp [hunk 1]
```diff
@@ -314,6 +314,8 @@ aditof::Status Adsd3500Sensor::open() {
             for (int i = 0; i < 10; i++) {
                 chipIDStatus = adsd3500_read_cmd(0x0112, &chipID);
                 if (chipIDStatus != Status::OK) {
+                    LOG(INFO) << "Could not read chip ID. Resetting ADSD3500 "
+                                 "to handle previous error.";
```
Comment (SeptimiuVana) [inline sdk/src/connections/target/adsd3500_sensor.cpp:318]: Can we add this message to the dealias check in the next if statement as well?

Comment (dNechita) [reply sdk/src/connections/target/adsd3500_sensor.cpp:318]: Is there a changes that it works to read chip ID but doesn't work to read dealias parameters?

Comment (SeptimiuVana) [reply sdk/src/connections/target/adsd3500_sensor.cpp:318]: There might be.

**Review by SeptimiuVana — APPROVED** [review]

File: sdk/src/connections/target/adsd3500_sensor.cpp (hunk 1 above)
Comment (dNechita) [reply sdk/src/connections/target/adsd3500_sensor.cpp:318]: I have added the message. Thanks!

**Pull request merged**